applications/serial_lte_modem/prj.conf

```diff
@@ -6,6 +6,8 @@
 # General config
 CONFIG_LOG=y
 CONFIG_LOG_DEFAULT_LEVEL=3
+CONFIG_MPU_STACK_GUARD=y
+#CONFIG_NO_OPTIMIZATIONS=y
```

Comment:
Why do this?

Reply:
It's added when we faced a MPU fault in adding the setting page. Thanks for the info, we'll remove it.
The OPTIMIZATION flag is added when we do GDB debugging the MPU Fault issue. It could be removed, too.